iceoryx_posh/test/moduletests/test_posh_runtime.cpp

```diff
@@ -258,6 +273,21 @@ TEST_F(PoshRuntime_test, GetMiddlewarePublisherIsSuccessful)
     EXPECT_EQ(publisherOptions.historyCapacity, publisherPort->m_chunkSenderData.m_historyCapacity);
 }
 
+TEST_F(PoshRuntime_test, MiddlewarePublisherMaxCapacityExceededReturnMaxCapacity)
```

Comment:
The testname is a little bit confusing. I would suggest writing
`GetMiddlewarePublisherWithHistoryGreaterMaxCapacityClampsHistoryToMaximum`.

Maybe a rule of thumb, the test name should describe what is happening from the user point of perspective.